crates/rpc/rpc/src/eth/pubsub.rs

```diff
@@ -147,10 +147,13 @@ where
                     Params::Bool(true) => {
                         // full transaction objects requested
                         let stream = pubsub.full_pending_transaction_stream().map(|tx| {
-                            EthSubscriptionResult::FullTransaction(Box::new(from_recovered(
-                                tx.transaction.to_recovered_transaction(),
-                                &tx_resp_builder,
-                            )))
+                            EthSubscriptionResult::FullTransaction(Box::new(
+                                from_recovered(
+                                    tx.transaction.to_recovered_transaction(),
+                                    &tx_resp_builder,
+                                )
+                                .expect("fill should be infallible"),
```

Comment:
this expect still needs to go

Reply:
Here also I've tried to log the error, otherwise it requires `TransactionCompat::Error : Serialize + Send`, which is something I don't know how to do.